examples/compose-bird/.idea/artifacts/compose_bird_js_1_0_0_alpha01.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ArtifactManager">
```

Comment:
Other examples have files in `.idea` directory ignored. Here is an example: https://github.com/theapache64/compose-jb/blob/compose-web-flappy-bird/examples/todoapp/.gitignore

Would be nice to keep them ignored for this example as well :)

Reply:
Yeah makes sense. Resolved in https://github.com/JetBrains/compose-jb/pull/797/commits/4749fa3bf0f3be87a9dc090aeff494bc36872644